tests/test.py

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/skeleton for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import unittest
+
+from ahocode import ahocode
+
+
+class test_automaton_methods(unittest.TestCase):

```

Comment:
I see these tests are from https://github.com/WojciechMula/pyahocorasick/blob/master/tests/test_basic.py. 

Please make a comment saying that the tests are from there with a link to the code.

Reply:
Thanks for this, i also added the proper link and repo name in ahocode.py.